nautobot/users/tests/test_password_ui.py

```diff
@@ -0,0 +1,79 @@
+from unittest import mock
+
+from django.contrib.auth import get_user_model
+from django.contrib.sessions.middleware import SessionMiddleware
+from django.test import override_settings, RequestFactory, TestCase
+from django.urls import reverse
+
+from social_django.utils import load_strategy, load_backend
+
+User = get_user_model()
+
+
+class PasswordUITest(TestCase):
+    def tearDown(self):
+        self.client.logout()
+
+    def test_change_password_enabled(self):
+        """
+        Check that a Django-authentication-based user is allowed to change their password
+        """
+        User.objects.filter(username="non_sso_user").delete()
```

Comment:
Is this necessary? `TestCase` should roll back any changes performed during the test.